packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/RealmObjectHelper.kt

```diff
@@ -1145,6 +1150,67 @@ internal object RealmObjectHelper {
         }
     }
 
+    @Suppress("unused") // Called from generated code
+    // Inlining this functions somehow break the IntelliJ debugger, unclear why?
+    internal fun realmToString(obj: BaseRealmObject): String {
```

Comment:
Would it make sense to iterate properties or is that to drastic?

Reply:
I was thinking about it, but I wanted to keep the default behavior smaller by default. Also with the Kotlin debugger you can actually see the underlying values (even though it requires you to click the getter).